### ISSUE TITLE
elm/parser |. operator

### ISSUE THREAD

**Issue by eimfach** [tonsky/FiraCode#908]
Hey there,

The font supports the `|=` from the elm/parser library but not `|.`

Would be nice if added, thanks.

**Comment by tonsky** [comment]
Yeah, that might be complicated. First because of possible conflicts, second with I’m not sure how to draw such a ligature. I’ll keep the issue around in case I decide to go back to it or make it optional. Thanks!
